hack/verify-terraform.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TERRAFORM_VERSION=0.12.29
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+function usage() {
+  echo >&2 "Usage: $0 <CLUSTER_NAME>"
+  exit 1
+}
+
+if [ $# != 1 ]; then
+    usage
+    exit 1
+fi
+
+TERRAFORM_ZIP="terraform_${TERRAFORM_VERSION}_linux_amd64.zip"
```

Comment:
I'd rather this work on my laptop too.  

I think a version check and erroring out if it doesn't exist/match would be a good start.  Let the developer work out how to get the correct version, and assume when this runs in a container it will have the correct version already installed

Handy for managing locally, maybe useful to allow CI to auto-install / switch:
- [`tfenv`](https://github.com/tfutils/tfenv) + `.terraform-version` files fit a pattern I know , but it's lots of bash
- [`tfswitch`](https://github.com/warrensbox/terraform-switcher) uses `version.tf` files which could be another option

Reply:
I'll go for `tfswitch` since it supports `version.tf`.